src/Illuminate/Cookie/CookieJar.php

```diff
@@ -30,6 +30,13 @@ class CookieJar implements JarContract
     protected $secure = false;
 
     /**
+     * The default httpOnly setting (defaults to true).
+     *
+     * @var bool
+     */
+    protected $httpOnly = true;
```

Comment:
OK that,s right . I,ll update the cookie service provider file to keep it as previously settings.